### ISSUE TITLE
i just have a question about learning rate

### ISSUE THREAD

**Issue by KareemSayed1232** [TranQuocTrinh/transformer#4]
does the lr uses the formula provided in the paper or not because i think i saw it some where but , now when i am searching for it i could not seem to find it anywhere

**Comment by tranquoctrinh** [comment]
@ksaid1232 Learning rate is a hyperparameter that is set based on experience.

**Comment by KareemSayed1232** [comment]
> @ksaid1232 Learning rate is a hyperparameter that is set based on experience.

okay thx <3

**Comment by ksaid1232** [comment]
thx <3
